website/releases/2025-06-11-3.0.0-release.md

```diff
@@ -0,0 +1,67 @@
+
+Learn more about [MLflow `LoggedModel`](https://mlflow.org/docs/latest/genai/data-model/#mlflow-loggedmodel-model-management) in the documentation.
+
+### 🔗 Strong Lineage Support
+
+Enhanced model tracking provides comprehensive lineage between models, runs, traces, and evaluation metrics. The new model-centric design allows you to group traces and metrics from interactive queries and automated evaluation jobs, enabling rich comparisons across model versions.
+
+<!-- ### Feedback Tracking
+
+Built-in assessment and feedback tracking capabilities allow you to capture both automated and human evaluation feedback directly tied to model executions and traces. This provides a comprehensive view of model performance across different evaluation dimensions. -->
+
+### 🧪 New GenAI Evaluation Suite
+
+MLflow 3.0 introduces a new evaluation suite for LLMs/GenAI. This new suite is only available in Managed MLflow on Databricks, but are coming soon to OSS MLflow. If you are interested in trying it out with free Databricks trial, click [here](https://docs.databricks.com/aws/en/mlflow3/genai/eval-monitor/) to learn more.
+
+### ⚡ Prompt Optimization
+
+The MLflow Prompt Registry now includes prompt optimization capabilities, allowing you to automatically improve prompts using evaluation feedback and labeled datasets. This includes versioning, tracking, and systematic prompt engineering workflows.
+
+Learn more about [prompt optimization](https://mlflow.org/docs/latest/genai/prompt-version-mgmt/prompt-registry/optimize-prompts) in the documentation.
+
+## Breaking Changes
+
+MLflow 3.0 includes several breaking changes as part of improving framework consistency and performance. Key changes include removal of MLflow Recipes, fastai and mleap flavors, and various deprecated API parameters.
+
+For the complete list of breaking changes, visit the [MLflow 3.0 Breaking Changes documentation](https://mlflow.org/docs/latest/ml/mlflow-3/breaking-changes).
+
+## Other Features
+
+**Streaming API for ResponsesAgent**: New streaming response capabilities through the ResponsesAgent class with predict_stream method, enabling real-time streaming for GenAI applications.
+
+**Auto-tracing support for Pydantic AI and `smolagents`**: Enhanced auto-tracing integrations for emerging GenAI frameworks, providing seamless observability out of the box.
+
+**Add `search_prompts` API for prompt registry**: New API functionality for searching and discovering prompts in the registry, making prompt management more efficient.
+
+**Support token tracking for OpenAI/LangChain auto-tracing**: Enhanced tracing now captures detailed token usage and cost information for better observability and cost management.
+
+**Record environment metadata in tracing**: Traces now capture comprehensive environment metadata including dependencies, system information, and execution context for better reproducibility.
```

Comment:
This is a lie.

Reply:
This refers to https://mlflow.org/docs/latest/genai/tracing/track-environments-context/. I'll update the contents.